src/browser/api_protocol/api_handlers/window.js

```diff
@@ -17,11 +17,10 @@ limitations under the License.
 const apiProtocolBase = require('./api_protocol_base');
 const Window = require('../../api/window').Window;
 const Application = require('../../api/application').Application;
-const _ = require('underscore');
 
-let successAck = {
-    success: true
-};
+// some local sugar
+const getWinId = message => apiProtocolBase.getTargetWindowIdentity(message.payload);
```

Comment:
`getWinId` might not be the best name as it is actually getting the target window (`getTargetWindowIdentity`) id and these can be different

Reply:
will rename to `getTargetWinId`